elasticsearch/src/main/scala/ElasticsearchServer.scala

```diff
@@ -18,7 +18,8 @@ class ElasticsearchServer {
 
   private val clusterName = "elasticsearch"
   private val dataDir = Files.createTempDirectory("elasticsearch_data_").toFile
-  private val settings = ImmutableSettings.settingsBuilder
+  private val settings = Settings.settingsBuilder
+    .put("path.home", dataDir.toString)
```

Comment:
The `path.home` config option is now required in v2.0+. See [this PR](https://github.com/elastic/elasticsearch/issues/13155) for some discussion.